mergify_engine/tests/unit/rules/test_rules.py

```diff
@@ -524,7 +524,7 @@ async def item(
     client.item.return_value = item()
     fake_repository.installation.client = client
 
-    fake_repository._caches.mergify_config.delete()
```

Comment:
don't we also need to keep this line like in the train tests ?

Reply:
No this test never goes through `fake_repository.get_mergify_config()`, it plays directly with `rules.get_mergify_config(file)`.